### ISSUE TITLE
Connecting to Web Server

### ISSUE THREAD

**Issue by camaro23** [dtag-dev-sec/tpotce#242]
# Contribution

Thank you for your decision to contribute to T-Pot.

## Issues

Hello,

When attempting to connect to the Web server, I am having trouble. Yesterday the IP address assigned to my Tpot was a 198.162.1.97 internal address. I was able to connect then. However, today Tpot shutdown and when I relaunched it I was provided with an IP of 75.49.124.66 and cannot connect to the web server. How can I change this to an internal address I can connect to? I am using a Bridged network.

Thanks!

Thank you :smiley:

-

<a name="faq"></a>
###### The honeypot logs are located in `/data/`. You have to login via ssh and run `sudo cd /data/`. Do not change any permissions here or T-Pot will fail to work.

-

<a name="info"></a>
### Baisc support information

- What T-Pot version are you currtently using?
- Are you running on a Intel NUC or a VM?
- How long has your installation been running?
- Did you install any upgrades or packages?
- Did you modify any scripts?
- Have you turned persistence on/off?
- How much RAM available (login via ssh and run `htop`)?
- How much stress are the CPUs under (login via ssh and run `htop`)?
- How much swap space is being used (login via ssh and run `htop`)?
- How much free disk space is available (login via ssh and run `sudo df -h`)?
- What is the current container status (login via ssh and run `sudo start.sh`)?

**Comment by t3chn0m4g3** [comment]
T-Pot will reboot every 24 hours and will be assigned an IP address according to your DHCP server settings. If you are accessing T-Pot via NAT you might want to think about using a DDNS service to account for IP address changes facing your internet side.

**Comment by camaro23** [comment]
I figured out the issue, Virtualbox was changing the MAC address. Changed to a static address. Is there a way to keep T-Pot permanently running?

**Comment by t3chn0m4g3** [comment]
Yes, you can modify `/etc/crontab` to your needs, but this will reduce security measures we take to ensure that in the unlikely event of a running container being compromised it will be wiped from RAM (reboot) and a fresh container being started from a clean image.

**Comment by camaro23** [comment]
Thank you. Not sure if you have heard of Wazuh, but if you have is there anyway to incorporate T-Pot alerts into Wazuh to have a standalone application for all alerting needs?

Thanks

**Comment by t3chn0m4g3** [comment]
Yes, you can modify `logstash.conf` and then use a new volume in `tpot.yml` to mount it into the logstash container. Will not go into detail here since it has been answered in the old issues. Please use the search function with mentioned keywords.

**Comment by camaro23** [comment]
What directory is logstash.conf in, and what is the best way to edit it?

**Comment by camaro23** [comment]
I have found logstah.conf and using the nano command to edit it. Do you have any example logstash.conf file and tpot.yml files of sending to a central agent? Do you have any videos demonstrating this type of configuration or teach a class? Sorry for being such a bother

**Comment by camaro23** [comment]
Or a way to send syslogs logs to a central Wazuh agent?

**Comment by camaro23** [comment]
in the Output section of the logstash.conf file. Can I edit the code to output elasticsearch to a host on the work?

output {
       elasticsearch {
            hosts => ["ipaddress:port']
          }

Thanks again

**Comment by LeoAral** [comment]
I can't connect to the web address, I get an error, can't connect to the server.
how can I fix it?
